crates/bitwarden/Cargo.toml

```diff
@@ -16,26 +16,28 @@ license-file.workspace = true
 [features]
 default = ["secrets"]
 
-secrets = [] # Secrets manager API
 internal = [
     "dep:bitwarden-exporters",
     "dep:bitwarden-generators",
 ] # Internal testing methods
+no-memory-hardening = [
+    "bitwarden-crypto/no-memory-hardening",
+] # Disable memory hardening features
 mobile = [
     "internal",
     "dep:uniffi",
     "bitwarden-crypto/mobile",
     "bitwarden-generators/mobile",
 ] # Mobile-specific features
-
+secrets = [] # Secrets manager API
 wasm-bindgen = ["chrono/wasmbind"]
 
 [dependencies]
 async-trait = ">=0.1.80, <0.2"
 base64 = ">=0.21.2, <0.23"
 bitwarden-api-api = { workspace = true }
 bitwarden-api-identity = { workspace = true }
-bitwarden-crypto = { workspace = true }
+bitwarden-crypto = { workspace = true, no-default-features = true }
```

Comment:
Should this not be `default-features = false`?  That's how it works outside of workspace dependencies at least
https://github.com/bitwarden/sdk/blob/68a0e4edda5d408ea2a499082c39879d03e87b9b/crates/bitwarden-uniffi/Cargo.toml#L26-L29

Weird that it doesn't give an error or warning either way.